data/test/news/articles/c0000000005o.json

```diff
@@ -134,77 +165,33 @@
                 }
               },
               {
-                "blockId": "bab00a0a-5148-2acc-5805-618026484615",
                 "type": "caption",
                 "model": {
                   "blocks": [
                     {
-                      "blockId": "44f0e618-46a9-37a9-eda4-8cefbb83fa3a",
                       "type": "text",
                       "model": {
                         "blocks": [
                           {
                             "type": "paragraph",
-                            "blockId": "6e8ba222-9bf7-114f-44d0-71ef73a04bc4",
                             "model": {
-                              "text": "**Suspendisse blandit tincidunt ante**, non finibus nisl ornare non."
-                            }
-                          }
-                        ]
-                      }
-                    }
-                  ]
-                }
-              }
-            ]
-          }
-        },
-        {
-          "type": "text",
-          "blockId": "8b851516-b175-4092-a398-692f84cd7268",
-          "model": {
-            "blocks": [
-              {
-                "blockId": "2996f344-7d91-90ec-9ba7-4b2f19677c3b",
-                "type": "paragraph",
-                "model": {
-                  "text": "Aliquam faucibus pulvinar condimentum. Duis sit amet arcu sit amet lectus maximus dignissim. Quisque ut vulputate lorem™."
-                }
-              }
-            ]
-          }
-        },
-        {
-          "blockId": "1475d1fb-5883-4c57-908a-2a89ba330ff6",
-          "type": "image",
-          "model": {
-            "blocks": [
-              {
-                "blockId": "13097b46-67c7-cbe6-5777-63088d479fea",
-                "type": "rawImage",
-                "model": {
-                  "width": 1024,
-                  "height": 576,
-                  "locator": "/cpsprodpb/D6C0/production/_101767945_gettyimages-519045912.jpg",
```

Comment:
Is there any reason this specific image has been removed?

Reply:
The diff is misleading, I haven't edited these files but replaced them with the fragmented payloads. I can check with @jimjohnsonrollings if he meant to remove this particular image

Reply:
I think that's just a mistake on my part - I've accidently removed an image block and a text block.

Reply:
Restored the text, image and text blocks

Reply:
FYI the locator was incorrectly prefixed with `/cpsprodpb/` so also removed that